frlg-any-percent-beginner/route.json

```diff
@@ -0,0 +1,13 @@
+{
+    "title": "FRLG Any% High Exp",
```

Comment:
These names appear to be flipped? Either that or the folder names are flipped.

Reply:
should be fixed